actionview/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Deprecate passing `nil` as value for the `model:` argument to the `form_with` method.
```

Comment:
Rather than adding a new one, I think you should just edit the changelog entry from https://github.com/rails/rails/pull/49943 since that hasn't been included in a Rails release yet.